### PR TITLE
chore: zwave-js bot

### PR DESCRIPTION
**Pull request by robertsLando**
@AlCalzone Is this enough?

### CODE REVIEW

File: .github/workflows/zwave-js-bot.yml [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+      - name: Check user's permissions to do this
+        id: check
+        uses: actions/github-script@v3
+        with:
+          github-token: ${{secrets.BOT_TOKEN}}
+          result-encoding: string
+          script: |
+            const bot = require(`${process.env.GITHUB_WORKSPACE}/.github/bot-scripts/index.js`);
+            return await bot.fixLintCheck({github, context});
```
Comment (AlCalzone) [inline .github/workflows/zwave-js-bot.yml:33]: You'll need to copy these files from my repo

File: .github/workflows/zwave-js-bot.yml [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+      - name: Give feedback
+        uses: actions/github-script@v3
+        env:
+          FEEDBACK: ${{ steps.fix.outputs.result }}
+        with:
+          github-token: ${{secrets.BOT_TOKEN}}
+          script: |
+            const bot = require(`${process.env.GITHUB_WORKSPACE}/.github/bot-scripts/index.js`);
+            return await bot.fixLintFeedback({github, context});
```
Comment (AlCalzone) [inline .github/workflows/zwave-js-bot.yml:120]: This one too

**Review by AlCalzone — CHANGES_REQUESTED** [review]
And if you want to try out the bot commenting on failed lints, I'll have to update the app repo.

File: .github/workflows/zwave-js-bot.yml (hunk 2 above)
Comment (AlCalzone) [reply .github/workflows/zwave-js-bot.yml:120]: Uhh that's the same, but you'll need these:
grafik

Comment (AlCalzone) [reply .github/workflows/zwave-js-bot.yml:120]: authorizedUsers.js defines who may execute the bot commands aside from the PR author:
grafik

File: .github/workflows/zwave-js-bot.yml (hunk 1 above)
Comment (robertsLando) [reply .github/workflows/zwave-js-bot.yml:33]: What if we create a `.github` repo in the org with them? Should it work?

Comment (robertsLando) [reply .github/workflows/zwave-js-bot.yml:33]: https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/creating-a-default-community-health-file

Comment (AlCalzone) [reply .github/workflows/zwave-js-bot.yml:33]: No, that runs the cloned scripts defined in your repo. Could maybe make it an action that we share between repos.

Comment (robertsLando) [reply .github/workflows/zwave-js-bot.yml:33]: Nope I don't think it will work, but maybe could be useful for other purposes :)

Comment (AlCalzone) [reply .github/workflows/zwave-js-bot.yml:33]: That doesn't mention scripts, but could come in handy.

File: .github/workflows/zwave-js-bot.yml [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+    types: [created] # edited, deleted
+
+jobs:
+  # #########################################################################
+  # Fix lint errors when an authorized person posts "@zwave-js-bot fix lint"
+  fix-lint:
+    if: |
+      contains(github.event.issue.html_url, '/pull/') &&
+      contains(github.event.comment.body, '@zwave-js-bot fix lint') &&
+      (github.event.comment.user.login != 'zwave-js-bot')
```
Comment (AlCalzone) [inline .github/workflows/zwave-js-bot.yml:14]: oh... might want to exclude `zwave-js-assistant[bot]` aswell.

Comment (AlCalzone) [reply .github/workflows/zwave-js-bot.yml:14]: https://github.com/zwave-js/node-zwave-js/commit/2fea442ce0485079f9d0f241c68056ef90a0324f

Comment (robertsLando) [reply .github/workflows/zwave-js-bot.yml:14]: PR opened thanks

**Pull request merged**